### ISSUE TITLE
Support for definining which files to copy

### ISSUE THREAD

**Issue by Hi-Fi** [droppedcode/typedoc-plugin-copy-assets#2]
Currently plugin copies all files that are referred as relative links (starting with ./ or ../). For images this works nicely and images are at the generated documentation. For other files like MDs or code examples coped files are not necessarily most user friendly.

[README](https://github.com/droppedcode/typedoc-plugin-copy-assets/blob/main/README.md?plain=1#L3) states that only images are copied, so I think it should be default action. If something else is needed, user could provide own filter. So default filtering value would be "all image extensions" (or normal ones), and if user would also want to get e.g. ts-files filter need to contain md and also those image extensions. So user given filter would fully override default one, not append to it.

**Comment by droppedcode** [comment]
Hi @Hi-Fi,
Thanks for the feature request.

It was added to version 1.0.5.

For complete functionality, please see the readme file.

In short, the default is to copy image references ```![label](./path/image.png)```. But you can enable all references with setting the ```onlyImages``` option. To fine tune what to include/exclude you can set regexes in the options.

```json
{
  "copyAssets": {
    "onlyImages": false,
    "include": [".(png|md)\\)$"],
    "exclude": ["\\(.*?exclude/.*?\\)$"]
  }
}
```

**Comment by droppedcode** [comment]
If you have any issues with this solution, please reopen the ticket.

**Comment by Hi-Fi** [comment]
For some reason I get error when trying to configure that copyAssets:
```
npx typedoc --logLevel Verbose
Loaded plugin @droppedcode/typedoc-plugin-copy-assets
[@knodes/typedoc-plugin-pages] ⇒ Using plugin version 0.23.1
Loaded plugin @knodes/typedoc-plugin-pages
Loaded plugin typedoc-plugin-replace-text
error Tried to set an option (copyAssets) that was not declared.
```

**Comment by droppedcode** [comment]
Can you check that npm installed the 1.0.5 version of the plugin?

**Comment by Hi-Fi** [comment]
I'm quite sure it was, that's why I was puzzled. Have to try with other project, too

**Comment by Hi-Fi** [comment]
```sh
> npm i -D @droppedcode/typedoc-plugin-copy-assets@latest

changed 2 packages, and audited 14 packages in 4s

found 0 vulnerabilities
npm notice 
npm notice New minor version of npm available! 8.5.5 -> 8.19.2
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.19.2
npm notice Run npm install -g npm@8.19.2 to update!
npm notice 
> npm run typedoc

> typedoc
> typedoc

Loaded plugin @droppedcode/typedoc-plugin-copy-assets
Loaded plugin typedoc-plugin-replace-text
error Tried to set an option (copyAssets) that was not declared.
```

**Comment by droppedcode** [comment]
I fixed the issue with 1.0.6

**Comment by Hi-Fi** [comment]
Thank you, now config doesn't complain at least anymore.
